network-api/networkapi/wagtailpages/templatetags/breadcrumbs.py

```diff
@@ -0,0 +1,17 @@
+from django import template
+
+from networkapi.wagtailpages.pagemodels.research_hub.landing_page import (
+    ResearchLandingPage,
+)
+
+register = template.Library()
+
+
+@register.inclusion_tag("fragments/breadcrumbs.html", takes_context=True)
+def get_research_breadcrumbs(context, include_self=False):
```

Comment:
I like that this is called ResearchBreadCrumbs, since they should point back to the ResearchLandingPage. 

If any other pages need to use breadcrumbs, we can add a method for them too, pointing back to whichever landing page we would like